docs/how_to/adding_multiple_sorting_interfaces.rst

```diff
@@ -68,12 +68,11 @@ Expected output:
 .. code-block:: text
 
     Units after adding first sorting:
-       unit_name
-    id
-    0          0
-    1          1
-    2          2
-    3          3
+    id  unit_name
```

Comment:
@weiglszonja mainly, it is fixing the headers so they are displayed as a single row here.

Reply:
ah ok, so just aesthetics